app/src/main/res/values-de/strings.xml

```diff
@@ -98,6 +98,7 @@
     <string name="sq_foot">Quadratfuß</string>
     <string name="sq_inch">Quadratzoll</string>
     <string name="acre">Acre</string>
+    <string name="are">Ar</string>
```

Comment:
It's *Ar* in German; *Are* seems to be only Swiss German according to Wikipedia.

Reply:
When building and running this pull request's branch locally, I see this when the language is set to English:

![screenshot_2018-07-01-15-55-57 1](https://user-images.githubusercontent.com/109998/42138731-b8e47dd6-7d47-11e8-8363-c1724b37ca32.png)


and this when the language is switched to German:

![screenshot_2018-07-01-15-56-12 1](https://user-images.githubusercontent.com/109998/42138732-c12c5c8e-7d47-11e8-92d5-75fbd91144ce.png)

Is that correct, or do I misunderstand the problem?

Reply:
Yep, that's correct – looks like I got a bit confused for some reason.